pages.fr/common/docker.md

```diff
@@ -0,0 +1,36 @@
+# docker
+
+> Gestion des conteneurs et des images Docker.
+> Plus d'informations : <https://docs.docker.com/engine/reference/commandline/cli/>.
+
+- Liste les conteneurs Docker en cours d'exécution :
```

Comment:
Why this space before the `:`?
```suggestion
- Liste les conteneurs Docker en cours d'exécution:
```

Reply:
This is a French rule : https://en.wikipedia.org/wiki/Colon_(punctuation)#Spacing ;)

Reply:
Oh, looks like we have a few french pages with that space.